src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -12128,6 +12191,726 @@ namespace AirflowNetwork {
         return AirLoopNumber;
     }
 
+    void Solver::DuctSizing()
+    {
+        using General::SolveRoot;
+        Real64 constexpr EPS(0.001);
+        int constexpr MaxIte(500);
+        Real64 constexpr MinVelocity(0.5);  // minimum airflow velocity (m/s)
+        Real64 constexpr MaxVelocity(20.0); // maximum airflow velocity (m/s)
+
+        int NodeLoopSupply = 0;
+        int NodeLoopReturn = 0;
+        int NodeSplitter = 0;
+        int NodeMixer = 0;
+        int NodeZoneIntlet = 0;
+        int NodeZoneReturn = 0;
+        int NumOfBranches = 0;
+        int AFNNodeNum;
+        int AFNLinkNum;
+        int AFNLinkNum1;
+        Real64 SumLength = 0.0;
+        Real64 DynamicLoss = 0.0;
+        Real64 MaxRough = 0.0;
+        bool DuctSizingSTFlag;
+        bool DuctSizingSBFlag;
+        bool DuctSizingRTFlag;
+        bool DuctSizingRBFlag;
+        Real64 hydraulicDiameter = 0.0;
+        Array1D<Real64> Par(9); // Parameters passed to RegulaFalsi
+        Real64 SupplyTrunkD = 0.0;
+        Real64 SupplyTrunkArea = 0.0;
+        Real64 SupplyBranchD = 0.0;
+        Real64 SupplyBranchArea = 0.0;
+        Real64 ReturnTrunkD = 0.0;
+        Real64 ReturnTrunkArea = 0.0;
+        Real64 ReturnBranchD = 0.0;
+        Real64 ReturnBranchArea = 0.0;
+        Real64 MdotBranch = 0.0;
+        int SolFla = 0;
+
+        int ZoneNum;
+        int NumOfCtrlZones = 0;
+        for (ZoneNum = 1; ZoneNum <= m_state.dataGlobal->NumOfZones; ++ZoneNum) {
```

Comment:
Scope reduction here and down below also.

Reply:
I define ZoneNum inside For Loop 

for (int ZoneNum = 1; ZoneNum <= m_state.dataGlobal->NumOfZones; ++ZoneNum) {